nautobot/extras/views.py

```diff
@@ -925,7 +925,12 @@ class ScheduledJobListView(generic.ObjectListView):
     table = tables.ScheduledJobTable
     filterset = filters.ScheduledJobFilterSet
     filterset_form = forms.ScheduledJobFilterForm
-    action_buttons = ()
+    action_buttons = ("delete",)
+
+
+class ScheduledJobBulkDeleteView(generic.BulkDeleteView):
+    queryset = ScheduledJob.objects.all()
+    table = tables.ScheduledJobTable
```

Comment:
Might need to also add `filterset = filters.ScheduledJobFilterSet` to this class?

Reply:
That seemed to not be an issue locally; remind me why would it be needed?

Reply:
IIRC if you filter the base table view, then "select all" via the checkbox in the top left, the filter needs to be propagated into the bulk-delete view so that it knows which "all" are selected.